frontend/src/components/GroupView/RightView/index.module.scss

```diff
@@ -5,7 +5,9 @@
 .RightView {
   display: flex;
   flex-direction: column;
-  height: 480px;
+  height: calc(100vh - 140px);
+  width: calc(100vw - 80px - 280px);
```

Comment:
We should move the width and height of group view stuff into a constant file, and directly reference that constant.

Reply:
Fixed this! Moved widths and heights into `groupview-sizes.scss`